{
    "expand": "operations,versionedRepresentations,editmeta,changelog,renderedFields",
    "fields": {
        "aggregateprogress": {
            "progress": 0,
            "total": 0
        },
        "aggregatetimeestimate": null,
        "aggregatetimeoriginalestimate": null,
        "aggregatetimespent": null,
        "assignee": null,
        "components": [{
            "description": "Tracks issues with build files.",
            "id": "12312593",
            "name": "Build Infrastructure",
            "self": "https://issues.apache.org/jira/rest/api/2/component/12312593"
        }],
        "created": "2008-12-10T07:48:10.000+0000",
        "creator": {
            "active": true,
            "avatarUrls": {
                "16x16": "https://issues.apache.org/jira/secure/useravatar?size=xsmall&avatarId=10452",
                "24x24": "https://issues.apache.org/jira/secure/useravatar?size=small&avatarId=10452",
                "32x32": "https://issues.apache.org/jira/secure/useravatar?size=medium&avatarId=10452",
                "48x48": "https://issues.apache.org/jira/secure/useravatar?avatarId=10452"
            },
            "displayName": "YihueyChyi",
            "key": "yihuey",
            "name": "yihuey",
            "self": "https://issues.apache.org/jira/rest/api/2/user?username=yihuey",
            "timeZone": "Etc/UTC"
        },
        "customfield_10010": null,
        "customfield_12310191": null,
        "customfield_12310192": null,
        "customfield_12310220": "2008-12-10T08:10:07.572+0000",
        "customfield_12310222": "1_*:*_1_*:*_648640874_*|*_5_*:*_1_*:*_0",
        "customfield_12310230": null,
        "customfield_12310250": null,
        "customfield_12310290": null,
        "customfield_12310291": null,
        "customfield_12310300": null,
        "customfield_12310310": "1.0",
        "customfield_12310320": null,
        "customfield_12310420": "66878",
        "customfield_12310920": "121975",
        "customfield_12310921": null,
        "customfield_12311020": null,
        "customfield_12311024": null,
        "customfield_12311120": null,
        "customfield_12311820": "0|i0l86f:",
        "customfield_12312022": null,
        "customfield_12312023": null,
        "customfield_12312024": null,
        "customfield_12312026": null,
        "customfield_12312220": null,
        "customfield_12312320": null,
        "customfield_12312321": null,
        "customfield_12312322": null,
        "customfield_12312323": null,
        "customfield_12312324": null,
        "customfield_12312325": null,
        "customfield_12312326": null,
        "customfield_12312327": null,
        "customfield_12312328": null,
        "customfield_12312329": null,
        "customfield_12312330": null,
        "customfield_12312331": null,
        "customfield_12312332": null,
        "customfield_12312333": null,
        "customfield_12312334": null,
        "customfield_12312335": null,
        "customfield_12312336": null,
        "customfield_12312337": null,
        "customfield_12312338": null,
        "customfield_12312339": null,
        "customfield_12312340": null,
        "customfield_12312341": null,
        "customfield_12312520": null,
        "customfield_12312521": "Wed Dec 17 19:58:50 UTC 2008",
        "customfield_12312720": null,
        "customfield_12312823": null,
        "customfield_12312920": null,
        "customfield_12312921": null,
        "customfield_12312923": null,
        "customfield_12313422": "false",
        "customfield_12313520": null,
        "description": "HIVEQL:  select l.http_user_agent,count(1) from log_resume_all l  group by l.http_user_agent\nMaybe I'll get error in the second stage:\n    FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.ExecDriver\n\nThe second stage :\nmap error \njava.lang.NullPointerException\n\tat org.apache.hadoop.hive.ql.exec.ReduceSinkOperator.process(ReduceSinkOperator.java:151)\n\tat org.apache.hadoop.hive.ql.exec.Operator.forward(Operator.java:250)\n\tat org.apache.hadoop.hive.ql.exec.MapOperator.process(MapOperator.java:174)\n\tat org.apache.hadoop.hive.ql.exec.ExecMapper.map(ExecMapper.java:71)\n\tat org.apache.hadoop.mapred.MapRunner.run(MapRunner.java:47)\n\tat org.apache.hadoop.mapred.MapTask.run(MapTask.java:219)\n\tat org.apache.hadoop.mapred.TaskTracker$Child.main(TaskTracker.java:2122)\n\nsyslog logs\n\n2008-12-10 15:41:15,209 DEBUG org.apache.hadoop.mapred.TaskTracker: Child starting\n2008-12-10 15:41:15,717 INFO org.apache.hadoop.metrics.jvm.JvmMetrics: Initializing JVM Metrics with processName=MAP, sessionId=\n2008-12-10 15:41:15,805 INFO org.apache.hadoop.mapred.MapTask: numReduceTasks: 64\n2008-12-10 15:41:16,252 INFO org.apache.hadoop.util.NativeCodeLoader: Loaded the native-hadoop library\n2008-12-10 15:41:16,253 INFO org.apache.hadoop.io.compress.zlib.ZlibFactory: Successfully loaded & initialized native-zlib library\n2008-12-10 15:41:16,424 INFO org.apache.hadoop.hive.ql.exec.MapOperator: Initializing Self\n2008-12-10 15:41:16,428 INFO org.apache.hadoop.hive.ql.exec.MapOperator: Adding alias /tmp/hive-root/462573742/46102483.10002 to work list for file /tmp/hive-root/462573742/46102483.10002/0015_r_000029_0\n2008-12-10 15:41:16,438 INFO org.apache.hadoop.hive.ql.exec.MapOperator: Got partitions: null\n2008-12-10 15:41:16,438 INFO org.apache.hadoop.hive.ql.exec.ReduceSinkOperator: Initializing Self\n2008-12-10 15:41:16,443 INFO org.apache.hadoop.hive.ql.exec.ReduceSinkOperator: Using tag = -1\n2008-12-10 15:41:16,460 INFO org.apache.hadoop.hive.serde2.thrift.TBinarySortableProtocol: Sort order is \"\"\n2008-12-10 15:41:16,460 INFO org.apache.hadoop.hive.serde2.thrift.TBinarySortableProtocol: Sort order is \"\"\n2008-12-10 15:41:16,489 INFO org.apache.hadoop.hive.ql.exec.MapOperator: DESERIALIZE_ERRORS:0\n2008-12-10 15:41:16,495 WARN org.apache.hadoop.mapred.TaskTracker: Error running child\njava.lang.NullPointerException\n\tat org.apache.hadoop.hive.ql.exec.ReduceSinkOperator.process(ReduceSinkOperator.java:151)\n\tat org.apache.hadoop.hive.ql.exec.Operator.forward(Operator.java:250)\n\tat org.apache.hadoop.hive.ql.exec.MapOperator.process(MapOperator.java:174)\n\tat org.apache.hadoop.hive.ql.exec.ExecMapper.map(ExecMapper.java:71)\n\tat org.apache.hadoop.mapred.MapRunner.run(MapRunner.java:47)\n\tat org.apache.hadoop.mapred.MapTask.run(MapTask.java:219)\n\tat org.apache.hadoop.mapred.TaskTracker$Child.main(TaskTracker.java:2122)\n\n\n\n\n\n",
        "duedate": null,
        "environment": "HADOOP 0.17.2.1 ",
        "fixVersions": [{
            "archived": false,
            "description": "released",
            "id": "12313637",
            "name": "0.3.0",
            "releaseDate": "2009-04-30",
            "released": true,
            "self": "https://issues.apache.org/jira/rest/api/2/version/12313637"
        }],
        "issuelinks": [],
        "issuetype": {
            "avatarId": 21133,
            "description": "A problem which impairs or prevents the functions of the product.",
            "iconUrl": "https://issues.apache.org/jira/secure/viewavatar?size=xsmall&avatarId=21133&avatarType=issuetype",
            "id": "1",
            "name": "Bug",
            "self": "https://issues.apache.org/jira/rest/api/2/issuetype/1",
            "subtask": false
        },
        "labels": [],
        "lastViewed": null,
        "priority": {
            "iconUrl": "https://issues.apache.org/jira/images/icons/priorities/major.svg",
            "id": "3",
            "name": "Major",
            "self": "https://issues.apache.org/jira/rest/api/2/priority/3"
        },
        "progress": {
            "progress": 0,
            "total": 0
        },
        "project": {
            "avatarUrls": {
                "16x16": "https://issues.apache.org/jira/secure/projectavatar?size=xsmall&pid=12310843&avatarId=11935",
                "24x24": "https://issues.apache.org/jira/secure/projectavatar?size=small&pid=12310843&avatarId=11935",
                "32x32": "https://issues.apache.org/jira/secure/projectavatar?size=medium&pid=12310843&avatarId=11935",
                "48x48": "https://issues.apache.org/jira/secure/projectavatar?pid=12310843&avatarId=11935"
            },
            "id": "12310843",
            "key": "HIVE",
            "name": "Hive",
            "projectCategory": {
                "description": "Scalable Distributed Computing",
                "id": "10292",
                "name": "Hadoop",
                "self": "https://issues.apache.org/jira/rest/api/2/projectCategory/10292"
            },
            "self": "https://issues.apache.org/jira/rest/api/2/project/12310843"
        },
        "reporter": {
            "active": true,
            "avatarUrls": {
                "16x16": "https://issues.apache.org/jira/secure/useravatar?size=xsmall&avatarId=10452",
                "24x24": "https://issues.apache.org/jira/secure/useravatar?size=small&avatarId=10452",
                "32x32": "https://issues.apache.org/jira/secure/useravatar?size=medium&avatarId=10452",
                "48x48": "https://issues.apache.org/jira/secure/useravatar?avatarId=10452"
            },
            "displayName": "YihueyChyi",
            "key": "yihuey",
            "name": "yihuey",
            "self": "https://issues.apache.org/jira/rest/api/2/user?username=yihuey",
            "timeZone": "Etc/UTC"
        },
        "resolution": {
            "description": "A fix for this issue is checked into the tree and tested.",
            "id": "1",
            "name": "Fixed",
            "self": "https://issues.apache.org/jira/rest/api/2/resolution/1"
        },
        "resolutiondate": "2008-12-17T19:58:50.000+0000",
        "status": {
            "description": "The issue is considered finished, the resolution is correct. Issues which are not closed can be reopened.",
            "iconUrl": "https://issues.apache.org/jira/images/icons/statuses/closed.png",
            "id": "6",
            "name": "Closed",
            "self": "https://issues.apache.org/jira/rest/api/2/status/6",
            "statusCategory": {
                "colorName": "green",
                "id": 3,
                "key": "done",
                "name": "Done",
                "self": "https://issues.apache.org/jira/rest/api/2/statuscategory/3"
            }
        },
        "subtasks": [],
        "summary": "group by count(1) will get error",
        "timeestimate": null,
        "timeoriginalestimate": null,
        "timespent": null,
        "updated": "2011-12-17T00:08:57.000+0000",
        "versions": [],
        "votes": {
            "hasVoted": false,
            "self": "https://issues.apache.org/jira/rest/api/2/issue/HIVE-150/votes",
            "votes": 0
        },
        "watches": {
            "isWatching": false,
            "self": "https://issues.apache.org/jira/rest/api/2/issue/HIVE-150/watchers",
            "watchCount": 0
        },
        "workratio": -1
    },
    "id": "12410345",
    "key": "HIVE-150",
    "self": "https://issues.apache.org/jira/rest/api/2/issue/12410345"
}